docs/src/main/paradox/pravega.md

```diff
@@ -0,0 +1,115 @@
+# Pravega
+
+[Pravega](http://www.pravega.io/) provides a new storage abstraction - a stream - for continuous and unbounded data. A Pravega stream is a durable, elastic, append-only, unbounded sequence of bytes that has good performance and strong consistency.
+
+@@project-info{ projectId="pravega" }
+
+## Artifacts
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-stream-alpakka-pravega_$scala.binary.version$
+  version=$project.version$
+}
+
+The table below shows direct dependencies of this module and the second tab shows all libraries it depends on transitively.
+
+@@dependencies { projectId="pravega" }
+
+
+## Concepts
+
+Pravega store at scale @extref[stream](pravega:pravega-concepts/#streams) of @extref[events](pravega:pravega-concepts/#events) organized within scopes.   
+
+Basically @extref[events](pravega:pravega-concepts/#streams) are appended in and read from @extref[streams](pravega:pravega-concepts/#streams).
+
+Pravega provides a durable storage with an unified API to access to cold events.
```

Comment:
Suggested rephrase:

> Pravega stores stream data durably, and applications can access the stream data using the same API both when tailing the stream and when processing past data. The system is architected so that the underlying storage is elastic and it is able to accommodate unbounded streams.